src/main/java/chess/domain/chessboard/ChessBoardGenerator.java

```diff
@@ -0,0 +1,67 @@
+package chess.domain.chessboard;
+
+import static chess.domain.chesspiece.Team.BLACK;
+import static chess.domain.chesspiece.Team.WHITE;
+
+import chess.domain.chesspiece.pawn.BlackPawn;
+import chess.domain.chesspiece.pawn.WhitePawn;
+import chess.domain.chesspiece.slidingPiece.Bishop;
+import chess.domain.chesspiece.Empty;
+import chess.domain.chesspiece.slidingPiece.King;
+import chess.domain.chesspiece.Knight;
+import chess.domain.chesspiece.pawn.Pawn;
+import chess.domain.chesspiece.Piece;
+import chess.domain.chesspiece.slidingPiece.Queen;
+import chess.domain.chesspiece.slidingPiece.Rook;
+import chess.domain.chesspiece.Team;
+import chess.domain.position.Position;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class ChessBoardGenerator {
+    private static final Map<Position, Piece> board = new LinkedHashMap<>();
+
+    public static Map<Position, Piece> initializeBoard() {
```

Comment:
```suggestion
    public static Map<Position, Piece> initializeBoard() {
```
(반영하지 않으셔도 됩니다.) 항상 `initializeBoard()`를 반환할거라 board를 들고 있는 필요가 없어보여 제거하고 그냥 메소드에서 바로 반환해주는 형태가 어떨까싶네요.

Reply:
생각해보니 그렇네요!! 불필요한 필드인 것 같습니다